### PR TITLE
Bugfix: don't change filter type when moving filter slider

### PR DESCRIPTION
**Pull request by ctzsnooze**
Fix for an issue in 1_44 / 4.3 where moving a filter slider would revert a changed filter type to the previously saved value.

To test, previous behaviour:
- change filter type (don't save)
- move slider -> filter type reverts to previously saved value

With this PR:
- change filter type (don't save)
- move slider -> filter type does not change.

### CODE REVIEW

**Review by KarateBrot — DISMISSED** [review]
Just compiled and tested it. I can confirm that it works and the filter type stays in place now.

**Comment by haslinghuis** [comment]
Actually it was a feature ;-) Filter type was a manual setting. Before we just disabled the sliders. Now we can have all possible combinations. So changing filter type needs a save for being stored in the pid_profile so when we move sliders it retrieves the stored value. But on second thought it doesn't need recalculation so we don't to update UI.

**Review by haslinghuis — APPROVED** [review]

File: src/js/TuningSliders.js
Comment (haslinghuis) [inline src/js/TuningSliders.js:755]: This block can be deleted in its entirety

```suggestion

```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
Please @ctzsnooze I have a small addition

There was a problem on initialization where we calculate the cutoffs in pid_tuning.js. Although it works for lp1 it does not for lp2.
From line 917 till line 1084 we have to change all TuningSliders values with FC.TUNING_SLIDERS values like so:
```
TuningSLiders.sliderGyroFilter should be replaced with FC.TUNING_SLIDERS.slider_gyro_filter
TuningSLiders.sliderGyroFilterMultiplier should be replaced with FC.TUNING_SLIDERS.slider_gyro_filter_multiplier
```
Same for DTerm.

File: src/js/TuningSliders.js
Comment (ctzsnooze) [reply src/js/TuningSliders.js:755]: Are you sure?
I noticed that earlier firmware versions will reset the slider type when disabling sliders due to manual entry. I have to say, this seems to be bad practice, but this is how it was.
So I thought we should not change the previous behaviour.
However I am happy to receive your further advice.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/js/TuningSliders.js
Comment (haslinghuis) [reply src/js/TuningSliders.js:755]: Previous firmware does not allow changing type. So sliders get disabled upon change.
As filters are not calculated this should have nothing to do with sliders

Comment (haslinghuis) [reply src/js/TuningSliders.js:755]: So 100% certain we can remove this

Comment (ctzsnooze) [reply src/js/TuningSliders.js]: Good, I have done so.

**Review by KarateBrot — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by chmelevskij — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Pull request merged**
